internal/meta/data.go

```diff
@@ -68,6 +68,7 @@ type Data struct {
 	Error         error         `meta:"-"`
 	json          map[string]string
 	exif          map[string]string
+	EmbeddedVideo string `meta:"EmbeddedVideo"`
```

Comment:
I'd move up e.g. between `Albums` and `Error` so the the private properties come last.